kolibri/content/api.py

```diff
@@ -24,6 +26,28 @@ class ChannelMetadataCacheViewSet(viewsets.ModelViewSet):
     def get_queryset(self):
         return models.ChannelMetadataCache.objects.all()
 
+    def destroy(self, request, pk=None):
```

Comment:
The UI to do this (the content management page) is only available to Device Owners. How do you place permissions on `destroy`?